presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorSessionProperties.java

```diff
@@ -36,6 +36,7 @@
     private static final String READER_MAX_MERGE_DISTANCE = "reader_max_merge_distance";
     private static final String READER_MAX_READ_SIZE = "reader_max_read_size";
     private static final String READER_STREAM_BUFFER_SIZE = "reader_stream_buffer_size";
+    private static final String MAX_BUFFER_SIZE = "max_buffer_size";
```

Comment:
`max_buffer_size` sounds too vague. The buffer is actually used in the page sink/writer, we can call it `writer_max_buffer_size`.

Reply:
Sure, I will make the change.

Reply:
Done.